Hamburger menu

AmineHLub opened the pull request:
<h1>Summary</h1>

- Added the new hamburger menu + Styled it
- Fixed minor different bugs (minor CSS html issues)
- Added JavaScript
- Added Linters

AmineHLub committed:
Add new hamburger-menu section

Style the new section

Fix bg images

Add link to both desktop and mobile

basnetlaxmi committed:
Add script.js file

basnetlaxmi committed:
Add Linter file

basnetlaxmi committed:
Fix Linter errors

AmineHLub committed:
Add box shadowing to hamberguer page menu

mricanho reviewed (changes requested):
Hi @AmineHLub, 

Good job so far! Your menu works as expected, and your linters set up is up and running🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Remove the padding-left of the ul and add some space between the li, do that and you are good to go, remember that you need to follow the original design, because the layout works visually. Double-check the position of the "X" too.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/131525278-684868f9-3989-4122-9558-653f98d9e484.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/131525370-b105dced-db4f-468c-b0fa-193e11a5e246.png)


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

![tuesday](https://user-images.githubusercontent.com/60631456/131525500-8581e747-0fe5-40c1-ab42-0026e51c655a.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmineHLub committed:
Fix padding left inside the ul > li

Fix the close X button position

Add margin between the menu lists

Make the design match figma

mikenath223 reviewed (approved):
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
